dataStandard_versioned_docs/version-5/whats-new/api_changes_5.md

```diff
@@ -0,0 +1,304 @@
+# API Changes from 5.1 to 5.2
+
+## New Endpoints
+
+### [Assessment Registration Domain](https://docs.ed-fi.org/reference/data-exchange/data-standard/model-reference/assessment-registration-domain/best-practices)
+
+#### assessmentAdministrations
+
+* **GET** /ed-fi/assessmentAdministrations
+    Retrieves specific assessment administrations using property values.
+* **POST** /ed-fi/assessmentAdministrations
+    Creates or updates assessment administrations based on natural key values.
+* **GET** /ed-fi/assessmentAdministrations/{id}
```

Comment:
All of these cases where we have `{id}` are causing Docusaurus's build process to fail. The `{ ... }` brackets are special syntax, which I did not notice previously.

We can solve this by making line 13 into an inline code block like this:

```suggestion
* `GET /ed-fi/assessmentAdministrations/{id}`
```